plugins/dependencies/src/main/kotlin/dependencies/JetBrains.kt

```diff
@@ -0,0 +1,31 @@
+@file:Suppress("PackageDirectoryMismatch", "SpellCheckingInspection", "unused")
+
+import de.fayard.refreshVersions.core.DependencyGroup
+import org.gradle.api.Incubating
+import org.gradle.kotlin.dsl.IsNotADependency
+
+@Incubating
+object JetBrains : IsNotADependency {
```

Comment:
I feel like Ktor could be added in here also.

They could be kept in different files for increased readability but add as val ktor = Ktor, etc

Reply:
@ConstantinCezarBegu you are totally right, done, thanks